nikola/nikola.py

```diff
@@ -2183,6 +2183,12 @@ def generic_page_renderer(self, lang, post, filters, context=None):
         context['title'] = post.title(lang)
         context['description'] = post.description(lang)
         context['permalink'] = post.permalink(lang)
+        if 'crumbs' not in context:
+            crumb_path = post.permalink(lang).lstrip('/').rstrip(self.config['INDEX_FILE'])
```

Comment:
rstrip doesn’t do what you think it does. The string is used as a list of characters to find and remove, not as a sequence. You need to remove it somehow else:

```python
crumb_path = post.permalink(lang).lstrip('/')
if crumb_path.endswith(self.config['INDEX_FILE']):
    crumb_path = crumb_path[:-len(self.config['INDEX_FILE'])]
```